ReleaseNotes.md

```diff
@@ -4,10 +4,76 @@ Back to [README.md](../README.md)
 
 ## Version
 
-2.18.8 STABLE
+2.18.9 STABLE
 
 ## Changes
 
+### 2.18.9
+
+   * dlt-user: fix crash with certain strings (#463)
+   * dlt_multiple_files: remove superfluous mode bits and add header file to header list (#462)
+   * Android: Add new feature in Android bp (#461)
+   * cmake: remove duplicated option message (#454)
+   * house-keeper: remove infinite wait (#438)
+   * dlt-logd-converter: Fix getting log level from log msg (#456)
+   * dlt-logd-converter: fixes android 12 compilation (#445)
+   * logfile: exhance internal dlt logging by introducing size limits (#369)
+   * This changes a mispatch from fcb676a7 to install the udp binary correctly. (#449)
+   * Installs dlt.conf on android (#446)
+   * dlt-connection: add socket timeout
+   * Fix memory leak
+   * Check for negative index in dlt_file_message
```

Comment:
Should add PR Ids:
This changes a mispatch from fcb676a7 to install udp binary correctly (#449)
Installs dlt.conf on android (#446)
dlt-connection: add socket timeout (#439)
Fix memory leak (#441)
Check for negative index in dlt_file_message (#437)